Dockerfile

```diff
@@ -37,6 +37,7 @@ COPY --from=builder /app/bin /app/bin
 WORKDIR /app/bin
 USER app
 
+ENV FXA_EMAIL_ENV production
 ENV ROCKET_ENV production
```

Comment:
I'm not sure we are using `ROCKET_ENV` for anything anymore, actually. I'll take a look.

Reply:
If ROCKET changes some internal functionality based on this we should keep it. Express in node.js suffers from this problem.